app.js

```diff
@@ -42,34 +43,62 @@ addMushroomButton.addEventListener('click', () => {
 
 addFriendButton.addEventListener('click', () => {
     // get the name from the input
+    const friendName = friendInputEl.value;
     // create a new friend object
+    const newFriend = {
+        name: friendName,
```

Comment:
Here we can use math.ceil and math.random to give our new friends a random name if we don't provide them a name.